### PR TITLE
feat(front-end): Created edit and manage organizations pages

### PR DESCRIPTION
**Pull request by NathanDimmer**

### CODE REVIEW

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        ? currentOrganizationData.organizationDescription
+        : ""
+    );
+  };
+
+  const saveOrganizationDescription = () => {
+    if (currentOrganizationData) {
+      setLoadingMessage("Updating Organization Description...");
+      updateOrganizationDescriptionDatabase(
+        currentOrganizationData.userId,
```
Comment (andrewdimmer) [inline front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx:54]: This needs to become organizationId.

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        organizationDescription
+      )
+        .then((value: any) => {
```
Comment (andrewdimmer) [inline front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx:57]: You shouldn't need to specify any... See if you can take the any out.

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+          if (value) {
+            setNotification({
+              type: "success",
+              message: "Organization Description Updated Successfully!",
+              open: true,
+            });
+            handleLoadOrganizationData(currentOrganizationData.userId);
```
Comment (andrewdimmer) [inline front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx:64]: This needs to become organizationId.

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationEmail.tsx [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+        ? currentOrganizationData.organizationEmail
+        : ""
+    );
+  };
+
+  const saveOrganizationEmail = () => {
+    if (currentOrganizationData) {
+      setLoadingMessage("Updating Organization Email...");
+      updateOrganizationEmailDatabase(
+        currentOrganizationData.userId,
```
Comment (andrewdimmer) [inline front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationEmail.tsx:54]: This needs to become organizationId.

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationEmail.tsx [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+        organizationEmail
+      )
+        .then((value: any) => {
```
Comment (andrewdimmer) [inline front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationEmail.tsx:57]: You shouldn't need to specify any... See if you can take the any out.

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationEmail.tsx [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+          if (value) {
+            setNotification({
+              type: "success",
+              message: "Organization Email Updated Successfully!",
+              open: true,
+            });
+            handleLoadOrganizationData(currentOrganizationData.userId);
```
Comment (andrewdimmer) [inline front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationEmail.tsx:64]: This needs to become organizationId.

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+        ? currentOrganizationData.organizationName
+        : ""
+    );
+  };
+
+  const saveOrganizationName = () => {
+    if (currentOrganizationData) {
+      setLoadingMessage("Updating Organization Name...");
+      updateOrganizationNameDatabase(
+        currentOrganizationData.userId,
```
Comment (andrewdimmer) [inline front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx:52]: This needs to become organizationId.

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+        organizationName
+      )
+        .then((value: any) => {
+          if (value) {
+            setNotification({
+              type: "success",
+              message: "Organization Name Updated Successfully!",
+              open: true,
+            });
+            handleLoadOrganizationData(currentOrganizationData.userId);
```
Comment (andrewdimmer) [inline front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx:62]: This needs to become organizationId.

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+        ? currentOrganizationData.organizationPhone
+        : ""
+    );
+  };
+
+  const saveOrganizationPhone = () => {
+    if (currentOrganizationData) {
+      setLoadingMessage("Updating Organization Phone...");
+      updateOrganizationPhoneDatabase(
+        currentOrganizationData.userId,
```
Comment (andrewdimmer) [inline front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx:54]: This needs to become organizationId.

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+        organizationPhone
+      )
+        .then((value: any) => {
+          if (value) {
+            setNotification({
+              type: "success",
+              message: "Organization Phone Updated Successfully!",
+              open: true,
+            });
+            handleLoadOrganizationData(currentOrganizationData.userId);
```
Comment (andrewdimmer) [inline front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx:64]: This needs to become organizationId.

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+  };
+
+  const saveOrganizationName = () => {
+    if (currentOrganizationData) {
+      setLoadingMessage("Updating Organization Name...");
+      updateOrganizationNameDatabase(
+        currentOrganizationData.userId,
+        organizationName
+      )
+        .then((value: any) => {
```
Comment (andrewdimmer) [inline front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx:55]: You shouldn't need to specify any... See if you can take the any out.

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx [hunk 12]
```diff
@@ -0,0 +1,174 @@
...
+  };
+
+  const saveOrganizationPhone = () => {
+    if (currentOrganizationData) {
+      setLoadingMessage("Updating Organization Phone...");
+      updateOrganizationPhoneDatabase(
+        currentOrganizationData.userId,
+        organizationPhone
+      )
+        .then((value: any) => {
```
Comment (andrewdimmer) [inline front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx:57]: You shouldn't need to specify any... See if you can take the any out.

File: front-end/src/Components/Layouts/EditOrganization.tsx [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+            >
+              <CloseIcon />
+            </IconButton>
+            <Typography variant="h6" className={classes.title}>
+              Edit organization
+            </Typography>
+          </Toolbar>
+        </AppBar>
+        <Container className={classes.pageTitle}>
+          <Typography variant="h3" id="profileTourStep1">
```
Comment (andrewdimmer) [inline front-end/src/Components/Layouts/EditOrganization.tsx:74]: Remove the left over id.

File: front-end/src/Components/Layouts/EditOrganization.tsx [hunk 14]
```diff
@@ -0,0 +1,129 @@
...
+            Profile
+          </Typography>
+        </Container>
+        <Container>
+          <Grid container spacing={6}>
+            <Grid item xs={12} sm={12} md={4} lg={3} id="profileTourStep10">
```
Comment (andrewdimmer) [inline front-end/src/Components/Layouts/EditOrganization.tsx:80]: Remove the left over id.

File: front-end/src/Components/Layouts/EditOrganization.tsx [hunk 15]
```diff
@@ -0,0 +1,129 @@
...
+              <CloseIcon />
+            </IconButton>
+            <Typography variant="h6" className={classes.title}>
+              Edit organization
+            </Typography>
+          </Toolbar>
+        </AppBar>
+        <Container className={classes.pageTitle}>
+          <Typography variant="h3" id="profileTourStep1">
+            Profile
```
Comment (andrewdimmer) [inline front-end/src/Components/Layouts/EditOrganization.tsx:75]: This should probably say something like "Organization Information", or something like that.

File: front-end/src/Components/Layouts/EditOrganization.tsx [hunk 16]
```diff
@@ -0,0 +1,129 @@
...
+interface EventInfoProps {
+  setNotification: (notification: NotificationMessage) => void;
+  currentOrganizationData: any;
+  setCurrentOrganizationData: any;
+  setLoadingMessage: any;
+  handleLoadOrganizationData: any;
+  classes: any;
+}
+
+const Organization: React.FunctionComponent<EventInfoProps> = ({
```
Comment (andrewdimmer) [inline front-end/src/Components/Layouts/EditOrganization.tsx:39]: Can we call this "EditOrganization" to match the file name?

File: front-end/src/Components/Layouts/EditOrganization.tsx [hunk 17]
```diff
@@ -0,0 +1,129 @@
...
+              />
+            </Grid>
+          </Grid>
+        </Container>
+      </Dialog>
+    </Fragment>
+  );
+};
+
+export default Organization;
```
Comment (andrewdimmer) [inline front-end/src/Components/Layouts/EditOrganization.tsx:129]: Can we call this "EditOrganization" to match the file name?

File: front-end/src/Components/Layouts/Organization.tsx [hunk 18]
```diff
@@ -0,0 +1,133 @@
...
+  Paper,
+} from "@material-ui/core";
+import React, { Fragment } from "react";
+import { NotificationMessage } from "../Misc/Notifications";
+import EditIcon from "@material-ui/icons/Edit";
+import EventIcon from "@material-ui/icons/Event";
+import DeleteIcon from "@material-ui/icons/Delete";
+interface EventInfoProps {
+  organizationData: any;
+  key: string;
```
Comment (andrewdimmer) [inline front-end/src/Components/Layouts/Organization.tsx:22]: key should not be a prop.

File: front-end/src/Components/Layouts/Organization.tsx [hunk 19]
```diff
@@ -0,0 +1,133 @@
...
+
+const Organization: React.FunctionComponent<EventInfoProps> = ({
+  setNotification,
+  key,
+  setEditOrganizationData,
+  setEventOrgId,
+  organizationData,
+  classes,
+}) => {
+  const style = useStyles();
```
Comment (andrewdimmer) [inline front-end/src/Components/Layouts/Organization.tsx:43]: This is now unused. Can we remove it to keep things clean?

File: front-end/src/Components/Layouts/Organization.tsx [hunk 20]
```diff
@@ -0,0 +1,133 @@
...
+interface EventInfoProps {
+  organizationData: any;
+  key: string;
+  setEventOrgId: Function;
+  setEditOrganizationData: Function;
+  setNotification: (notification: NotificationMessage) => void;
+  classes: any;
+}
+
+const useStyles = makeStyles({
```
Comment (andrewdimmer) [inline front-end/src/Components/Layouts/Organization.tsx:29]: This is now unused. Can we remove it to keep things clean?

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx (hunk 1 above)
Comment (NathanDimmer) [reply front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx:54]: Done

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx (hunk 2 above)
Comment (NathanDimmer) [reply front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx:57]: Done

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx (hunk 3 above)
Comment (NathanDimmer) [reply front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx:64]: Done

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationEmail.tsx (hunk 4 above)
Comment (NathanDimmer) [reply front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationEmail.tsx:54]: Done

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationEmail.tsx (hunk 6 above)
Comment (NathanDimmer) [reply front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationEmail.tsx:64]: Done

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx (hunk 7 above)
Comment (NathanDimmer) [reply front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx:52]: Done

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx (hunk 11 above)
Comment (NathanDimmer) [reply front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx:55]: Done

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx (hunk 8 above)
Comment (NathanDimmer) [reply front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationName.tsx:62]: Done

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx (hunk 9 above)
Comment (NathanDimmer) [reply front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx:54]: Done

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx (hunk 12 above)
Comment (NathanDimmer) [reply front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx:57]: Done

File: front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx (hunk 10 above)
Comment (NathanDimmer) [reply front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx:64]: Done

File: front-end/src/Components/Layouts/EditOrganization.tsx (hunk 16 above)
Comment (NathanDimmer) [reply front-end/src/Components/Layouts/EditOrganization.tsx:39]: Done

File: front-end/src/Components/Layouts/EditOrganization.tsx (hunk 13 above)
Comment (NathanDimmer) [reply front-end/src/Components/Layouts/EditOrganization.tsx:74]: Done

File: front-end/src/Components/Layouts/EditOrganization.tsx (hunk 15 above)
Comment (NathanDimmer) [reply front-end/src/Components/Layouts/EditOrganization.tsx:75]: Done

File: front-end/src/Components/Layouts/EditOrganization.tsx (hunk 14 above)
Comment (NathanDimmer) [reply front-end/src/Components/Layouts/EditOrganization.tsx:80]: Done

File: front-end/src/Components/Layouts/EditOrganization.tsx (hunk 17 above)
Comment (NathanDimmer) [reply front-end/src/Components/Layouts/EditOrganization.tsx:129]: Done

File: front-end/src/Components/Layouts/Organization.tsx (hunk 18 above)
Comment (NathanDimmer) [reply front-end/src/Components/Layouts/Organization.tsx:22]: Done

File: front-end/src/Components/Layouts/Organization.tsx (hunk 20 above)
Comment (NathanDimmer) [reply front-end/src/Components/Layouts/Organization.tsx:29]: Done

File: front-end/src/Components/Layouts/Organization.tsx (hunk 19 above)
Comment (NathanDimmer) [reply front-end/src/Components/Layouts/Organization.tsx:43]: Done

**Pull request merged**